tests/integration/generate_fixtures/go_ethereum.py

```diff
@@ -361,6 +361,50 @@ def setup_chain_state(w3):
     return geth_fixture
 
 
+def update_doc_version(new_version):
+    file_path = "./docs/contributing.rst"
+    with open(file_path) as file:
+        lines = file.readlines()
+
+    replaced = False
+    for i, line in enumerate(lines):
+        if "python -m geth.install " in line:
+            lines[i] = f"      $ python -m geth.install {new_version}\n"
```

Comment:
These are missing the `v` which is necessary but looks good otherwise 👌🏼 

```suggestion
            lines[i] = f"      $ python -m geth.install v{new_version}\n"
```